src/components/EventSignUp/index.tsx

```diff
@@ -60,173 +62,181 @@ const EventSignUp: React.FC<Props> = ({ id }) => {
         }
     };
 
+    const handleGroupCountChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        // sets count to 1 if field is cleared
+        setGroupCount(parseInt(e.target?.value) || 1);
+    };
+
     const handleInputMaskChange = (e: React.ChangeEvent<HTMLInputElement>) => {
         // eslint-disable-next-line @typescript-eslint/unbound-method
         e.preventDefault;
         setPhoneNumber(e.target?.value);
     };
 
+    const groupEvent = () => {
+        if (groupSignUp == true) {
+            return (
+                <input
+                    type="number"
+                    name="groupCount"
+                    onChange={handleGroupCountChange}
+                    placeholder="Group Count (Optional)"
+                    className={styles.otherInput}
+                    id="groupCountField"
+                />
+            );
+        }
+    };
```

Comment:
Here you can do ```{ groupSignUp && <input ... /> }``` for conditional rendering. Not worth changing now but just wanted to let you know of an easier way.

Reply:
Ah, thanks for the tip!